### PR TITLE
feat: Update submit photo screen

### PR DESCRIPTION
**Pull request by lrupnikk**

### CODE REVIEW

File: brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/submitphoto/SubmitPhotoFragment.kt [hunk 1]
```diff
@@ -73,12 +73,17 @@ class SubmitPhotoFragment : Fragment(),
             loadingView.isVisible = state.loadingVisible
 
             val isSelfie = state.documentType == DocumentType.SELFIE
-            ivSelfie.isVisible = isSelfie
-            ivDocument.isVisible = !isSelfie
+            val content1 =
+                if (isSelfie) R.string.SubmitPhoto_Selfie_CheckedItem1 else R.string.SubmitPhoto_Selfie_CheckedItem2
+            val content2 =
+                if (isSelfie) R.string.SubmitPhoto_Document_CheckedItem1 else R.string.SubmitPhoto_Document_CheckedItem2
```
Comment (roknovak) [inline brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/submitphoto/SubmitPhotoFragment.kt:79]: First string id is not correct

File: brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/submitphoto/SubmitPhotoFragment.kt [hunk 2]
```diff
@@ -73,12 +73,17 @@ class SubmitPhotoFragment : Fragment(),
             loadingView.isVisible = state.loadingVisible
 
             val isSelfie = state.documentType == DocumentType.SELFIE
-            ivSelfie.isVisible = isSelfie
-            ivDocument.isVisible = !isSelfie
+            val content1 =
+                if (isSelfie) R.string.SubmitPhoto_Selfie_CheckedItem1 else R.string.SubmitPhoto_Selfie_CheckedItem2
```
Comment (roknovak) [inline brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/submitphoto/SubmitPhotoFragment.kt]: Wrong implementation for content1 and content2. It should be:

```
val content1 =
                if (isSelfie) R.string.SubmitPhoto_Selfie_CheckedItem1 else R.string.SubmitPhoto_Document_CheckedItem1
val content2 =
                if (isSelfie) R.string.SubmitPhoto_Selfie_CheckedItem2 else R.string.SubmitPhoto_Document_CheckedItem2
```

File: brd-android/fabriik-common/src/main/res/values/text_styles.xml [hunk 3]
```diff
@@ -29,6 +29,7 @@
 
     <style name="TextStyle.Title6">
         <item name="android:textSize">16sp</item>
+        <item name="lineHeight">20dp</item>
```
Comment (roknovak) [inline brd-android/fabriik-common/src/main/res/values/text_styles.xml]: Can we remove lineHeight attribute?

File: brd-android/fabriik-common/src/main/res/values/text_styles.xml [hunk 4]
```diff
@@ -50,7 +51,7 @@
     <style name="TextStyle.Body2">
         <item name="android:textSize">14sp</item>
         <item name="fontFamily">@font/roboto_regular</item>
-        <item name="lineHeight">20dp</item>
+        <item name="lineSpacing">6dp</item>
```
Comment (roknovak) [inline brd-android/fabriik-common/src/main/res/values/text_styles.xml]: Wrong property

**Review by roknovak — APPROVED** [review]

**Review by dabitdev — APPROVED** [review]

**Pull request merged**